package.json

```diff
@@ -58,13 +58,15 @@
   ],
   "scripts": {
     "suffixup": "curl -o public_suffix_list.dat https://publicsuffix.org/list/public_suffix_list.dat && ./generate-pubsuffix.js",
-    "test": "vows test/*_test.js"
+    "test": "vows test/*_test.js",
+    "coverage": "node node_modules/istanbul/lib/cli.js cover vows test/*_test.js"
```

Comment:
npm adds `./node_modules/.bin` directory to PATH. I think, after local installation of istanbul you'll find `./node_modules/.bin/istanbul`, so you'll be able to use just `istanbul` in your npm scripts.